try.scss

```diff
@@ -0,0 +1,11 @@
+.foo {
+  .bar {
+    .baz {
+      .spam {
+        .ham {
```

Comment:
Empty rule<br>Nesting should be no greater than 4, but was 5<br>Selector should have depth of applicability no greater than 2, but was 5